daemon/resize.go

```diff
@@ -37,5 +38,17 @@ func (daemon *Daemon) ContainerExecResize(name string, height, width int) error
 	if err != nil {
 		return err
 	}
-	return daemon.containerd.ResizeTerminal(context.Background(), ec.ContainerID, ec.ID, width, height)
+	// Wait for the exec session ready.
+	timeout := 10 * time.Second
+	c := make(chan struct{})
```

Comment:
since we end up making a channel here to allow for the timeout, may as well make `ReadyWaitGroup` into a channel like originally suggested by @tonistiigi

Reply:
@tonistiigi @mlaventure I just learned that a closed channel would always be able to poll from. (Do feel have learned a lot about go programming just for this PR.) I will make code changes to use a chan to indicate the readiness.  Thanks for the suggestions.